src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/Helpers.cs

```diff
@@ -23,6 +23,8 @@ internal static RoslynTrigger GetRoslynTrigger(AsyncCompletionData.CompletionTri
         {
             switch (trigger.Reason)
             {
+                case AsyncCompletionData.CompletionTriggerReason.InvokeAndCommitIfUnique:
+                    return new RoslynTrigger(CompletionTriggerKind.InvokeAndCommitIfUnique);
```

Comment:
major :'( that we didn't have a test of this behavior until now.  BUt great that you've fixed it!